src/extensions/default/bramble/lib/ConsoleManagerRemote.js

```diff
@@ -0,0 +1,49 @@
+(function(transport) {
+    "use strict";
+
+    function _log(s){
+        transport.send("bramble-console", s);
+    }
+
+    // Create Console if not present
+    if (!window.console) {
+        window.console = ({});
+    }
+
+    var console = (window.console);
+
+    // Implement Console Log
+    if (!console.log) {
+        console.log = ({});
+    }
+
+    // Bind _log to iframe console.
+    console.log = _log;
+
+    // Implement other Log Levels to console.log
+    if (!console.debug) {
```

Comment:
For all these other `console.*` methods we *always* want to re-bind them to `console.log` and therefore `_log` so drop all these `if` checks.